components/Layout/DatePicker.tsx

```diff
@@ -35,12 +68,13 @@ export default function HeaderDatePicker({
         onChange={(newValue) => {
           if (dayjs(newValue).isValid()) datePickerSetValue(newValue)
         }}
+        inputFormat={getDateFormat()} // Moved inputFormat here
```

Comment:
This is fine, just can you please remove the comment.

Reply:
Fixed in https://github.com/TUM-Dev/TUMenu/pull/110/commits/972b7d8ca37be9e33adeda3e0367e6b305a155f4